doc/source/whatsnew/v0.25.0.rst

```diff
@@ -37,7 +67,7 @@ Other API Changes
 
 - :class:`DatetimeTZDtype` will now standardize pytz timezones to a common timezone instance (:issue:`24713`)
 - ``Timestamp`` and ``Timedelta`` scalars now implement the :meth:`to_numpy` method as aliases to :meth:`Timestamp.to_datetime64` and :meth:`Timedelta.to_timedelta64`, respectively. (:issue:`24653`)
--
+- :meth:`Timestamp.strptime` will now rise a NotImplementedError (:issue:`21257`)
```

Comment:
leftover?

Reply:
I moved this entry to `Other API Changes` as it was floating where the sub-sections should be.